### ISSUE TITLE
re.compile with a sufficiently Unicode-y regex fails due to loading Python array module

### ISSUE THREAD

**Issue by eddieantonio** [brython-dev/brython#892]
**Brython version**: 3.6.2
**Browser**: Chrome 67.0.3396.99

Minimal Brython code:

```html
<html>
<head>
    <script type="text/javascript" src="/Brython-3.6.2/brython.js"></script>
    <script type="text/javascript" src="/Brython-3.6.2/brython_stdlib.js"></script>
</head>
<body onload="brython()">
   <script type="text/python">
import re
pat = re.compile(r"""\b(?:(?:[ptkcshmny]w?|w)?[aâāi'îīoôōeêē](?:[hs]?[ptkcmn]|h|s|y|w)?|r|l)+\b""")
  </script>
</body>
</html>
```

The underlying issue seems to be that the array module is missing from the standard distribution, but this is what I'm trying to do when I get the error.

Relevant log: [127.0.0.1-1531934532532.log](https://github.com/brython-dev/brython/files/2206726/127.0.0.1-1531934532532.log):
![log](https://user-images.githubusercontent.com/2294397/42897622-c4691e3a-8a7d-11e8-8bbf-c3f2c4a9a1f7.png)

**Comment by PierreQuentel** [comment]
Does the code really fails ? In my version of Brython 3.6.2 I get the same messages in the console, but they are not errors (cf the section on 404 errors in the [FAQ](http://www.brython.info/static_doc/en/faq.html)) and the regular expression is compiled and usable.

**Comment by eddieantonio** [comment]
Sorry, you're right! I think what happened is that I encountered #900 and #901 at the same time as  making this code, and debugging the actual cause of my issues was difficult due to the many "false positive" error messages. I may have been using a different version of the released version of Brython 3.6.2, or the latest master release. My mistake! Closing.

**Comment by PierreQuentel** [comment]
No worries :-)
